application/src/main/java/org/togetherjava/tjbot/commands/tags/TagCommand.java

```diff
@@ -32,24 +35,33 @@ public TagCommand(TagSystem tagSystem) {
 
         this.tagSystem = tagSystem;
 
-        // TODO Thing about adding an ephemeral selection menu with pagination support
+        // TODO Think about adding an ephemeral selection menu with pagination support
         // if the user calls this without id or similar
-        getData().addOption(OptionType.STRING, ID_OPTION, "the id of the tag to display", true);
+        getData().addOption(OptionType.STRING, ID_OPTION, "the id of the tag to display", true)
+            .addOption(OptionType.USER, MENTION_USER_OPTION,
+                    "optionally, the user who you want to reply to", false);
     }
 
     @Override
     public void onSlashCommand(@NotNull SlashCommandEvent event) {
         String id = Objects.requireNonNull(event.getOption(ID_OPTION)).getAsString();
+        OptionMapping mentionUserOption = event.getOption(MENTION_USER_OPTION);
+
         if (tagSystem.handleIsUnknownTag(id, event)) {
             return;
         }
 
-        event
+        ReplyAction message = event
             .replyEmbeds(new EmbedBuilder().setDescription(tagSystem.getTag(id).orElseThrow())
                 .setFooter(event.getUser().getName() + " • used " + event.getCommandString())
                 .setTimestamp(Instant.now())
                 .setColor(TagSystem.AMBIENT_COLOR)
-                .build())
-            .queue();
+                .build());
+        if (mentionUserOption != null) {
+            message = message.setContent(mentionUserOption.getAsMember().getAsMention());
+        }
```

Comment:
You could rewrite this as
```java
event.replyEmbeds(...)
  .setFooter(...)
  ...
  .setContent(mentionUserOption == null ? null : mentionUserOption.getAsmember().getAsMention())
  .build()
.queue();
```
(i.e. a single method call chain without the variable and if-else interruption)

debatable though if this is more readable. Just wanting to give u the option.